requirements.txt

```diff
@@ -1,4 +1,4 @@
 dill>=0.2.7.1
 pytest>=3.0.7
-reportportal-client>=5.0.6
-six>=1.15.0
\ No newline at end of file
+reportportal-client>=5.0.7
```

Comment:
@rplevka @iivanou Can we use this PR to bring in 5.0.8 release that just went out?